cmd/kubeadm/app/util/config/masterconfig.go

```diff
@@ -56,9 +56,17 @@ func SetInitDynamicDefaults(cfg *kubeadmapi.InitConfiguration) error {
 	// This is the same logic as the API Server uses
 	ip, err := netutil.ChooseBindAddress(addressIP)
 	if err != nil {
-		return err
+		switch err.(type) {
```

Comment:
Shall we make it to a private function, since it is used in both masterconfig and nodeconfig.